spec/functions/datetime.md

```diff
@@ -0,0 +1,313 @@
+## Date and Time Value Formatting
+
+This subsection describes the functions and options for date/time formatting.
+Selection based on date and time values is not required in this release.
+
+> [!NOTE]
+> Selection based on date/time types is not required by MF2.
+> Implementations should use care when defining selectors based on date/time types.
```

Comment:
This uses a normative word ("should") as if it were not normative... in a non-normative note and dances around the word "required". I'd suggest instead:

```suggestion
> This version of MF2 does not define selection on date/time types.
> Use care if trying to provide implementation-specific support for this feature.
```